### ISSUE TITLE
gridx not appearing in migrated project

### ISSUE THREAD

**Issue by JonFerraiolo** [maqetta/maqetta#2415]
I was assigned the gridx tests, and I was running them on my migrated M5 project. To my surprise, the widget palette doesn't show gridx!

I'm pretty sure it's because <myproject>/.settings/libs.settings is preserved from M5. It seems like we need to add a gridx entry to existing libs.settings files.

@childsb @aerwin @peller

**Comment by jhpedemonte** [comment]
I already have this in the migration script.  Should have added a new line for the 'gridx' library.

Can you please check the `libs.settings` file in your project?

**Comment by JonFerraiolo** [comment]
I figured out what's wrong. I have an eclipse project, which inserts an extra `WebContent/` folder. The gridx line is in my libs.settings file but doesn't have the correct path to the gridx library. As a result, the widget palette doesn't include gridx.

**Comment by jhpedemonte** [comment]
Odd. The migration script should have handled that.  Check your Eclipse project in the jferrai account.  Ping me later with the account and project you see this in.

**Comment by JonFerraiolo** [comment]
This is weird. I swear I logged into the staging server and looked at libs.settings for my Eclipse project and it didn't show WebContent/ for the gridx entry. I verified the problem on both Mac/FF and Win/FF - no gridx widget in Widgets palette. But today that file is correct and I see gridx in the Widgets palette.

So, we should probably close this. Either I messed up somehow (although I don't see how - I verified the problem from a couple of angles) and/or an older libs.settings file was cached
